__tests__/calendar-restful-model-collection-spec.js

```diff
@@ -166,6 +166,60 @@ describe('CalendarRestfulModelCollection', () => {
     });
   });
 
+  test('[CONSECUTIVE AVAILABILITY] should fetch results with params', done => {
+    const params = {
+      startTime: '1590454800',
+      endTime: '1590780800',
+      interval: 5,
+      duration: 30,
+      emails: [{ name: 'Jane', email: 'jane@email.com' }],
```

Comment:
currently the API just accepts lists of lists of emails, e.g.,
```
[["jane@email.com" ],["bill@email.com","ted@email.com"]]
```
looks for consecutive slots where jane is free and biill and ted are both free

Reply:
@tbadun Ah gotcha, I think the documentation is wrong since that's where I got this from: https://developer.nylas.com/docs/api/#post/calendars/availability/consecutive

So just curious, will the first array (jane) always only have one email where as the second array has all the other participants we want to check against?

Reply:
yeah, the docs are a bit off unfortunately, I flagged it with doc team a while ago but should follow up.

within the array of arrays, you can have as many arrays as you want, each sub-array will contain as many emails as are required to check for a given meeting. 

```
"emails": [["jane@email.com" ],["bill@email.com"],["ted@email.com"]]
```
will look for 3 consecutive timeslots: one with jane, one with bill one with ted (arbitrary order)

```
"emails": [["jane@email.com" ,"bill@email.com","ted@email.com"]]
```
will look for 1 timeslot where jane, bill, and ted are all available

basically, any sub-array will check for the availability of just those individuals.

Reply:
Ahhhh perfect explanation, thanks!